component/commons/handler/pom.xml

```diff
@@ -0,0 +1,157 @@
+            <artifactId>org.apache.felix.scr.ds-annotations</artifactId>
+        </dependency>
+        <dependency>
+            <groupId>org.jacoco</groupId>
+            <artifactId>org.jacoco.agent</artifactId>
+            <classifier>runtime</classifier>
+        </dependency>
+        <dependency>
+            <groupId>org.testng</groupId>
+            <artifactId>testng</artifactId>
+        </dependency>
+        <dependency>
+            <groupId>org.powermock</groupId>
+            <artifactId>powermock-module-testng-common</artifactId>
+        </dependency>
+        <dependency>
+            <groupId>org.powermock</groupId>
+            <artifactId>powermock-api-mockito</artifactId>
+        </dependency>
+        <dependency>
+            <groupId>org.mockito</groupId>
+            <artifactId>mockito-all</artifactId>
+        </dependency>
+        <dependency>
+            <groupId>org.powermock</groupId>
+            <artifactId>powermock-module-testng</artifactId>
+        </dependency>
+        <dependency>
+            <groupId>org.wso2.carbon.identity.framework</groupId>
+            <artifactId>org.wso2.carbon.identity.event</artifactId>
+        </dependency>
+
+    </dependencies>
+
+    <build>
+        <plugins>
+            <plugin>
+                <groupId>org.apache.felix</groupId>
+                <artifactId>maven-scr-plugin</artifactId>
+            </plugin>
+            <plugin>
+                <groupId>org.apache.felix</groupId>
+                <artifactId>maven-bundle-plugin</artifactId>
+                <extensions>true</extensions>
+                <configuration>
+                    <instructions>
+                        <Bundle-SymbolicName>${project.artifactId}</Bundle-SymbolicName>
+                        <Bundle-Name>${project.artifactId}</Bundle-Name>
+                        <Private-Package>org.wso2.carbon.identity.password.expiry.validation.handler.internal
+                        </Private-Package>
+                        <Export-Package>
+                            !org.wso2.carbon.identity.password.expiry.validation.handler.internal,
+                            org.wso2.carbon.identity.password.expiry.validation.handler.*
+                        </Export-Package>
+                        <Import-Package>
+                            org.wso2.carbon.identity.mgt.*; version="${carbon.identity.package.import.version.range}",
+                            org.apache.commons.logging.*; version="1.0.4",
+                            org.osgi.framework,
+                            org.osgi.service.component.*,
+                            org.wso2.carbon.identity.application.authentication.framework.*,
+                            javax.servlet,
+                            javax.servlet.http,
+                            org.wso2.carbon.databridge.commons; version="${carbon.analytics-common.version}",
+                            *;resolution:=optional
```

Comment:
Shall we avoid from adding wild card import packages and specify only the required packages? Additionally, it would be better if we can add the version/version ranges.